policyengine_us/parameters/gov/states/va/tax/income/spouse_tax_adjustment/adjustment_limit.yaml

```diff
@@ -0,0 +1,15 @@
+description: Virginia cannot provide spouse tax adjustment that exceeds this amount.
+values: 
+  2021-01-01: 259
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Virginia spouse tax adjustment
+  reference:
+  - title: Virginia 2021 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2021-760-instructions.pdf#page=20
+  - title: Virginia 2022 Form 760 Resident Individual Income Tax Booklet
+    href: https://www.tax.virginia.gov/sites/default/files/vatax-pdf/2022-760-instructions.pdf#page=19
+  - title: Virginia Department of Taxation
```

Comment:
need to find the legal code on this website:
https://law.lis.virginia.gov/vacode/title58.1/chapter3/  

it has to be somewhere